week-2/I-array-properties/exercise.js

```diff
@@ -6,7 +6,13 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  if (arr.length == 0) {
+    return "empty";
+  } else {
+    return "notEmpty";
+  }
+
```

Comment:
This is doing the right thing, but we want it to return a boolean value (`true` or `false`), not a string - see the comment showing the "Expected Result" from running the program.